CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -22,7 +22,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Fix [close temp file(s) in case an error happens during defragmentation](https://github.com/etcd-io/etcd/pull/18854)
 
 ### Dependencies
-- Compile binaries using [go 1.22.9](https://github.com/etcd-io/etcd/pull/18849).
+- Compile binaries using [go 1.22.10](https://github.com/etcd-io/etcd/pull/19004).
```

Comment:
Please add a dependencies section in v3.5.18 rather than changing v3.5.17, which has already been released.

Reply:
I have made the changes. Please review them.